AddOns/BasicFinancials/app/src/codeunit/AssistedSetup.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
+{
+    Access = Internal;
+
+    var
+        AssistedSetupTxt: Label 'Set up the Basic Financials extension';
```

Comment:
I believe that this should be "Set up the Basic Financials"

Reply:
Brent has previous commented on a pull request. That it should be "Set up the Basic Financials extension"